.eslintrc

```diff
@@ -1,3 +1,16 @@
 {
-  "extends": ["frost-standard"]
+  "extends": ["frost-standard"],
+  "rules": {
+    "ocd/sort-import-declarations": [
+      1,
```

Comment:
May as well make this a `2` and make it error for this project, since you already fixed them.